[Bug]: TS2304 compilation errors

### Bug description

Steps to reproduce the problem:

1. upgrade to 18.0.1
2. compile


### Puppeteer version

18.0.1

### Node.js version

18.9.0

### npm version

8.19.1

### What operating system are you seeing the problem on?

Linux

### Relevant log output

```shell
.../puppeteer-core/lib/types.d.ts(2879,48): error TS2304: Cannot find name 'Poller_2'.
.../puppeteer-core/lib/types.d.ts(5697,13): error TS2552: Cannot find name 'Poller_2'. Did you mean 'Poller'?
.../puppeteer-core/lib/types.d.ts(5698,13): error TS2304: Cannot find name 'MutationPoller'.
.../puppeteer-core/lib/types.d.ts(6012,39): error TS2304: Cannot find name 'createDeferredPromise'.
.../puppeteer-core/lib/types.d.ts(6032,43): error TS2304: Cannot find name 'Poller_2'.
.../puppeteer-core/lib/types.d.ts(6367,18): error TS2304: Cannot find name 'isSuitableNodeForTextMatching'.
.../puppeteer-core/lib/types.d.ts(6368,18): error TS2304: Cannot find name 'TextContent_2'.
.../puppeteer-core/lib/types.d.ts(6368,18): error TS2552: Cannot find name 'TextContent_2'. Did you mean 'TextContent'?
.../puppeteer-core/lib/types.d.ts(6369,18): error TS2304: Cannot find name 'createTextContent'.
```

--- a/src/injected/injected.ts
+++ b/src/injected/injected.ts
@@ -15,23 +15,35 @@
  */
 
 import {createDeferredPromise} from '../util/DeferredPromise.js';
-import * as Poller from './Poller.js';
-import * as TextContent from './TextContent.js';
+import {RAFPoller, MutationPoller, IntervalPoller} from './Poller.js';
+import {
+  isSuitableNodeForTextMatching,
+  createTextContent,
+} from './TextContent.js';
 import * as TextQuerySelector from './TextQuerySelector.js';
 import * as XPathQuerySelector from './XPathQuerySelector.js';
 import * as PierceQuerySelector from './PierceQuerySelector.js';
 import * as util from './util.js';
 
+/**
+ * @internal
+ */
 const PuppeteerUtil = Object.freeze({
   ...util,
-  ...Poller,
-  ...TextContent,
   ...TextQuerySelector,
   ...XPathQuerySelector,
   ...PierceQuerySelector,
   createDeferredPromise,
+  createTextContent,
+  IntervalPoller,
+  isSuitableNodeForTextMatching,
+  MutationPoller,
+  RAFPoller,
 });
 
+/**
+ * @internal
+ */
 type PuppeteerUtil = typeof PuppeteerUtil;
 
 /**

--- a/test-d/puppeteer.test-d.ts
+++ b/test-d/puppeteer.test-d.ts
@@ -6,7 +6,7 @@ import {
   executablePath,
   launch,
   default as puppeteer,
-} from '../lib/esm/puppeteer/puppeteer.js';
+} from '..';
 
 expectType<typeof launch>(puppeteer.launch);
 expectType<typeof connect>(puppeteer.connect);

